cves/kernel/CVE-2021-3764.yml

```diff
@@ -116,7 +118,7 @@ upvotes_instructions: |
   upvotes to each vulnerability you see. Your peers will tell you how
   interesting they think this vulnerability is, and you'll add that to the
   upvotes score on your branch.
-upvotes:
+upvotes: 2
```

Comment:
+1 to the upvotes, because while it was simple, it is interesting to see how little mistakes can have big impacts.